packages/camera/camera_web/example/integration_test/camera_web_test.dart

```diff
@@ -1664,7 +1664,7 @@ void main() {
             'with notFound error '
             'if the camera does not exist', (WidgetTester tester) async {
           expect(
-            () async => await CameraPlatform.instance.getMaxZoomLevel(
+            () async => CameraPlatform.instance.getMaxZoomLevel(
```

Comment:
is await optional here? are these functions still async?

Reply:
Everything is still async; see https://dart-lang.github.io/linter/lints/unnecessary_await_in_return.html for a full example of the lint that these changes are fixing.

Reply:
fascinating!